crates/sui-types/src/storage.rs

```diff
@@ -56,7 +56,16 @@ pub enum DeleteKind {
 #[derive(Debug)]
 pub enum ObjectChange {
     Write(Object, WriteKind),
+    /// NOTE: UnwrapThenDelete will only show up in Delete when protocol config feature flag
+    /// `disable_wrapped_tombstone` is false.
+    /// When the flag is true, UnwrapThenDelete will be in a separate entry below.
```

Comment:
Thoughts on stamping this enum out? So
```
Delete(SequenceNumber),
Wrap(SequenceNumber),
UnwrapThenDeleteDEPRECATED(SequenceNumber),
UnwrapThenDelete(SequenceNumber),
```
?
I just worry of the potential of bugs to be introduced that accidentally uses the old, deprecated path